IPython/utils/ulinecache.py

```diff
@@ -10,35 +8,9 @@
 from IPython.utils import py3compat
 from IPython.utils import openpy
 
-if py3compat.PY3:
-    getline = linecache.getline
-    
-    # getlines has to be looked up at runtime, because doctests monkeypatch it.
-    @functools.wraps(linecache.getlines)
-    def getlines(filename, module_globals=None):
-        return linecache.getlines(filename, module_globals=module_globals)
```

Comment:
that can likely be reduced further, as now the wrapper is no-op, we can just search for the usage, and use `linecache.getlines` directly. We can leave this function there with a deprecation warning.